event-handlers/Emergence/People/RegistrationRequestHandler/registerComplete/listshine.php

```diff
@@ -0,0 +1,9 @@
+<?php
+//this file should be in /event-handlers/People/RegistretationRequestHandler/registerComplete/  directory of your site
+if(isset($_POST['Subscribe'])){
```

Comment:
@themightychris Yes there is, it checks if there is an api key inserted, if there is it will show it:
`
{if RemoteSystems\ListShine::$api_key}
                    <div class="checkbox">  
                        <label>  
                            <input type="checkbox" id="Subscribe" name="Subscribe">I want to receive emails  
                        </label>          
                    </div>  
{/if}  
`

Should I edit the code so it checks if there is an api key in listshine.conf.php before running? That model sounds good